composer.json

```diff
@@ -1,30 +1,30 @@
 {
     "name": "setono/sylius-terms-plugin",
-    "type": "sylius-plugin",
     "description": "Sylius terms and conditions plugin",
+    "license": "MIT",
+    "type": "sylius-plugin",
     "keywords": [
         "sylius",
         "sylius-plugin",
         "terms"
     ],
-    "license": "MIT",
     "require": {
-        "php": ">=7.4",
+        "php": ">=7.4 || ^8.0",
         "behat/transliterator": "^1.3",
         "doctrine/collections": "^1.6",
         "doctrine/orm": "^2.7",
         "knplabs/knp-menu": "^3.1",
         "sylius/resource-bundle": "^1.6",
-        "symfony/config": "^4.4 || ^5.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0",
-        "symfony/form": "^4.4 || ^5.0",
-        "symfony/http-foundation": "^4.4 || ^5.0.0",
-        "symfony/http-kernel": "^4.4 || ^5.1.5",
-        "symfony/options-resolver": "^4.4 || ^5.0",
-        "symfony/routing": "^4.4 || ^5.0",
+        "symfony/config": "^4.4 || ^5.0 || ^6.0",
+        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
+        "symfony/form": "^4.4 || ^5.0 || ^6.0",
+        "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
+        "symfony/http-kernel": "^4.4 || ^5.1.5 || ^6.0",
+        "symfony/options-resolver": "^4.4 || ^5.0 || ^6.0",
+        "symfony/routing": "^4.4 || ^5.0 || ^6.0",
         "symfony/translation-contracts": "^1.1 || ^2.4",
         "thecodingmachine/safe": "^1.1",
-        "twig/twig": "^2.14",
+        "twig/twig": "^2.14 || ^3.3",
```

Comment:
Is 3.3 needed for PHP8+ or why do you add this? :)

Reply:
Sylius 1.12 also supports twig ^3.3